1904-12-19.xml

```diff
@@ -0,0 +1,4093 @@
+                        <row>
+                            <cell>Dunville &amp; Co, Ltd.</cell>
+                            <cell>Belfast.</cell>
+                            <cell>Old Irish Whiskies.</cell>
+                        </row>
+                        <row>
+                            <cell>Wm. Lanahan &amp; Son.</cell>
+                            <cell>Baltimore.</cell>
+                            <cell>Monongshels XXXX Whiskey.</cell>
+                        </row>
+                        <row>
+                            <cell>The Cook &amp; Bernheimer Co.</cell>
+                            <cell>New York.</cell>
+                            <cell>Old Valley Whiskey and Gold Lion Cocktails.</cell>
+                        </row>
+                        <row>
+                            <cell>Stone &amp; Son.</cell>
+                            <cell>London.</cell>
+                            <cell>Guinness' Stout &amp; Bass' Pale Ale.</cell>
+                        </row>
+                        <row>
+                            <cell>Freund Ballor &amp; Co.</cell>
+                            <cell>Torino.</cell>
+                            <cell>Vermouth.</cell>
+                        </row>
+                        <row>
+                            <cell>Pierre Bisset.</cell>
+                            <cell>Cette.</cell>
+                            <cell>Vermouth &amp; Aperitives.</cell>
+                        </row>
+                        <row>
+                            <cell>Terrabonatea Company, Ld.</cell>
+                            <cell><hi rend="bold"/></cell>
+                            <cell>Teas.</cell>
+                        </row>
+                    </table>
+                    <p>Depot for Prince Metternich's "Richardsquelle," the best mineral table water
+                        in the world.</p>
+                    <p>Great assortment of Wines, Spirits, Liqueurs, of the finest Brands, etc</p>
+                    <p> EVOLD STELLA </p>
+                    <p>D.C. Scotch whisky</p>
+                    <p>LEES COMPANY</p>
+                    <p>ELINDUNGI</p>
+                    <p>Bots Agents for $gypt &amp; Sadas G. G. DROSSOS &amp; CO</p>
+                    <p> LATE PRAZZICA &amp; DROSSOS. ALEXANDRIA. CAIRO, PONT-SAID AND TUBO </p>
+                    <p> KBARZOUM 87.6.908 </p>
+                </div>
+                <!--This was not templated^-->
+
+                <div type="item">
+                    <p> BOEHME &amp; ANDERER NEXT DOOR TO TURF CLUB, </p>
+                    <p>
+                        <hi rend="italic">CAIRO</hi>
+                    </p>
+                    <p>Xmas Exhibition.</p>
+                    <p>Great Selection</p>
+                    <p>of Gifts for the Season.</p>
+                    <p>7.8.95</p>
+                    <!--This was not templated^-->
+                </div>
+                <div type="item">
```

Comment:
this item contains a whole bunch of adverts, which should be siblings of the advert that precedes it. If you remove the `<div type="item">` tag and the `</div>` that closes it at line 1605, it will fix it.